frappe/public/js/frappe/form/controls/link.js

```diff
@@ -471,7 +471,7 @@ frappe.ui.form.ControlLink = class ControlLink extends frappe.ui.form.ControlDat
 			docname: value,
 			fields: columns_to_fetch,
 		}).then((response) => {
-			if (!response || !response.name) return "";
+			if (!response || !response.name) return undefined;
```

Comment:
Nitpick:
```suggestion
			if (!response || !response.name) return null;
```

ideally, application code should not set or respond with "undefined". null can be used in such cases.

> explicit absence of value is better represented with `null`. `undefined` means a variable has been declared without a value assignment.